central/reports/scheduler/v2/schedule.go

```diff
@@ -203,6 +210,14 @@ func newSchedulerImpl(reportConfigDatastore reportConfigDS.DataStore, reportMeta
 /* Concurrency and scheduling functions */
 
 func (s *scheduler) Start() {
+	ctx := sac.WithGlobalAccessScopeChecker(context.Background(),
+		sac.AllowFixedScopes(
+			sac.AccessModeScopeKeys(storage.Access_READ_ACCESS, storage.Access_READ_WRITE_ACCESS),
+			sac.ResourceScopeKeys(resources.WorkflowAdministration)))
+
+	s.queuePendingReports(ctx)
+	s.queueScheduledReports(ctx)
+	go s.runReports()
```

Comment:
start 2 instances of the `s.runReports()` func?

Reply:
Oops that's a mess-up when trying to address comments.

Reply:
+1